apps/docs/content/getting-started/examples.mdx

```diff
@@ -6,10 +6,11 @@ date: 10/11/2023
 order: 3
 ---
 
-Here's a [very simple sandbox](https://codesandbox.io/s/tldraw-example-canary-2lrzmy) showing the tldraw component.
+Below is a [simple sandbox](https://codesandbox.io/s/tldraw-example-canary-2lrzmy) showing the tldraw component.
 
-<Embed src="https://codesandbox.io/embed/2lrzmy?view=Editor+%2B+Preview&module=%2Fsrc%2Fapp.tsx" />
+The [tldraw examples app](https://github.com/tldraw/tldraw/blob/main/apps/examples/src) contains a lot of helpful examples showing how to customise the UI, use the editor API, handle assets, create custom shapes/tools and much more. However, the examples app is best experienced via cloning and running the [tldraw repo](https://github.com/tldraw/tldraw) locally.
```

Comment:
Grab me offline and maybe let's talk about this in person: I think we should rethink that there's copy here in general, really. We want more UI around this.

There should be a big main call-to-action - [See our examples] which is a link to https://examples.tldraw.com/

After that big CTA, there should just be an organized list:
- Framework examples: Next.js, Vite, CRA
- Examples source: source code for examples listed above
- Multiplayer implementation: adds collaboration via Yjs

And _really_ we want this list to go away and be more consolidated in general with the examples app in the end.

Again, let's talk more in person!